genie-agent/src/main/java/com/netflix/genie/agent/cli/AgentOptionsArgumentsImpl.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+import com.beust.jcommander.Parameter;
+import org.springframework.stereotype.Component;
+
+/**
+ * Implementation of AgentOptions delegate.
+ */
+@Component
+public class AgentOptionsArgumentsImpl implements ArgumentDelegates.AgentOptions {
+
+    @Parameter(
+        names = {"--no-cleanup"},
+        description = "Don't remove the job folder"
```

Comment:
I would say something more like "Don't delete anything from disk in the job folder upon job completion". Cause aren't there two options that don't remove the job folder? This and the default?